packages/block/src/blockchainMachine.ts

```diff
@@ -41,15 +41,15 @@ export class BlockchainMachine<TBlock extends IBlockStub> extends StartStopServi
 
     protected async startInternal(): Promise<void> {
         if (!this.blockProcessor.started) this.logger.error("The BlockProcessor should be started before the BlockchainMachine.");
-        if (!this.actionStore.started) this.logger.error("The ActionStore should be started before the BlockchainMachine.");
+        if (!this.actionStore.started) this.logger.error("The CachedKeyValueStore should be started before the BlockchainMachine.");
```

Comment:
The message should probably still say actionStore, if that's what we call the property

Reply:
Fixed in f805a6e.